server/src/__tests__/completions.test.ts

```diff
@@ -1259,4 +1259,60 @@ describe('On Completion', () => {
       ])
     )
   })
+
+  it('Provide proper completion items on LICENSE variable', async () => {
+    analyzer.analyze({
+      uri: DUMMY_URI,
+      document: FIXTURE_DOCUMENT.COMPLETION
+    })
+
+    const resultBeforeResolve = onCompletionHandler({
+      textDocument: {
+        uri: DUMMY_URI
+      },
+      position: {
+        line: 31,
+        character: 17
+      }
+    })
+
+    const licenseTested = 'GPL-2.0-with-bison-exception'
+
+    const expectedResultBeforeResolve = {
+      label: licenseTested,
+      kind: 12,
+      deprecated: true,
+      labelDetails: {
+        description: 'Source: SPDX License List'
+      },
+      documentation: 'Loading...',
+      textEdit: {
+        range: {
+          start: { line: 31, character: 11 },
+          end: { line: 31, character: 28 }
+        },
+        newText: licenseTested
+      }
+    }
+
+    expect(resultBeforeResolve).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining(expectedResultBeforeResolve)
+      ])
+    )
+
+    const completionItem = resultBeforeResolve.find((item) => item.label === licenseTested)
+
+    if (completionItem === undefined) {
+      throw new Error('completionItem is undefined')
```

Comment:
Why not use expect.isDefined? It will appear as a propper test failure rather than register an uncaught exception

Reply:
Otherwise Typescript will complain `completionItem` is undefined when calling `onCompletionResolveHandler(completionItem)`.

The only alternative I can think of would be something like
``` typescript
expect(completionItem).toBeDefined()
if (completionItem === undefined) {
  return
}
```

which is equally ridiculous, but one line longer.

In all cases, this is somewhat a pointless check since the previous `expect(resultBeforeResolve)` already checked there is an item with that label. If that line is ever reached, I'd consider it an error in the test itself, not in the code that is tested

Reply:
Ok